community/kernel/src/main/java/org/neo4j/kernel/impl/locking/community/CommunityLockClient.java

```diff
@@ -42,6 +42,18 @@ public CommunityLockClient( LockManagerImpl manager )
     }
 
     @Override
+    public Locks.Client description( String desc )
+    {
+        return this;
+    }
```

Comment:
The community lock manager prints the resource access graph on deadlock, so it's much more helpful already. Agree that these descriptions would be really nice in community lock manager as well though, vote we do that in a follow up commit at some point